### PR TITLE
Some ROI-style discussion points from the roundtable

### PR DESCRIPTION
**Pull request by robmoffat**
for discussion

### CODE REVIEW

File: docs/goals-objectives/resources.md [hunk 1]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
 
 Placeholder for OSR GitHub issue https://github.com/finos/open-source-readiness/issues/55
+
+_This is based on the OSFF Roundtable discussion on OSPO ROI_
+
+How do OSPOs measure their effectiveness?  This seems to be an open area for discussion in the finance industry at the moment.  The list below summarizes some of the thinking around this.
+
+## Talent Acquisition
+
+- Does having an Open-Source presence materially affect hiring talent?  Is there a reflected benefit in working on Open-Source?
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:15]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 2]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+
+## Code Consumption
+
+- Can you measure the amount of Open-Source code the organisation is consuming?
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:19]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 3]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+How do OSPOs measure their effectiveness?  This seems to be an open area for discussion in the finance industry at the moment.  The list below summarizes some of the thinking around this.
+
+## Talent Acquisition
+
+- Does having an Open-Source presence materially affect hiring talent?  Is there a reflected benefit in working on Open-Source?
+
+## Code Consumption
+
+- Can you measure the amount of Open-Source code the organisation is consuming?  
+- Is there a way of justifying the OSPO based on risk-reduction around consuming Open-Source code?
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:20]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 4]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+- Does consuming Open-Source reduce the amount of proprietary code that an institution uses?  In which case, is there a cost-saving on license fees for using Open-Source?
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:21]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 5]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+
+## De-Duplication
+
+- Often, an Open-Source library will contain _bugs_, discovered by developers internal to your organisation.  The default (only) option available is to take ownership of a copy of the library and maintain bug fixes in your copy.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:25]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 6]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+
+- Can you measure the amount of Open-Source code the organisation is consuming?  
+- Is there a way of justifying the OSPO based on risk-reduction around consuming Open-Source code?
+- Does consuming Open-Source reduce the amount of proprietary code that an institution uses?  In which case, is there a cost-saving on license fees for using Open-Source?
+
+## De-Duplication
+
+- Often, an Open-Source library will contain _bugs_, discovered by developers internal to your organisation.  The default (only) option available is to take ownership of a copy of the library and maintain bug fixes in your copy.   
+- This quickly becomes onerous when there are multiple teams within the organisation doing this for the same libraries.  
+- There is a strong case for allowing Open-Source contribution in order to lighten the load of this maintenance and ownership.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:27]: `Open-Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 7]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+- It should be possible to look at where an institution has contributed fixes to Open Source projects - measuring this might be a way to justify allowing Open Source contribution, since each time this happens you are avoiding the above scenario happening.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:28]: `Open Source` should be `open source`

File: docs/goals-objectives/resources.md [hunk 8]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+- This can also happen not just for bugs but for missing functionality.
+
+## External Contribution
+
+- Where an institution open-sources its own code, how much contribution occurs from outside the organisation?
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:33]: `Open-Sources` should be `open sources`

File: docs/goals-objectives/resources.md [hunk 9]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+- For firms like [Facebook](https://facebook.com), open-sourcing [React](https://github.com/facebook/react) has lead to the creation of an entire ecosystem based around this software.  Facebook benefit from all this post-hoc innovation for free.
+- At the moment, financial institutions don't really have a React-level project, but nevertheless it would be useful to track external contribution.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:35]: `React-level` should be `React level`

File: docs/goals-objectives/resources.md [hunk 10]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+- For a project like [Perspective](), originally donated by JP Morgan, contributions are now being made both from committers external to JP Morgan and also alumni.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:36]: Perspective link missing

File: docs/goals-objectives/resources.md [hunk 11]
```diff
@@ -5,3 +5,44 @@ sidebar_label: Defining Corporate Objectives
...
+
+## Popularity
+
+- Metrics like stars and forks on GitHub repos go some way towards measuring this.
+
+## Strategic Objectives
+
+- Often a firm will have strategic objectives which dovetail into open-source.  For example, if your firm wishes to publish APIs then the OSPO should be recognised as a part of achieving these.
```
Comment (mcleo-d) [inline docs/goals-objectives/resources.md:44]: `open-source` should be `open source`

**Review by mcleo-d — CHANGES_REQUESTED** [review]
Hi @robmoffat - There are some minor changes needed to the text before merging. Hopefully straightforward. Thanks for the hard work here!

**Review by psmulovics — APPROVED** [review]

**Pull request merged**